docs/getting_started/configuration.rst

```diff
@@ -38,7 +38,8 @@ environment variable can be provided on the command line like so:
 
 .. code-block:: bash
 
-   HTTPS_PROXY=https://localhost:3128 ./prawbot.py
+   HTTPS_PROXY=https://localhost:3128
+   ./prawbot.py
```

Comment:
Why are these on separate lines?

Reply:
These are separate commands. I assume you set the variable, and then run the command.

Reply:
Fair assumption, however, it's not quite correct. Environment variables can be set on a per-command basis inline as originally done. They can also be set for the duration of the parent program (terminal) but that must be done like so:

```sh
export VARIABLE=value
```

Since we only intend to set it for the child process (python), we provide the environment variable on the same line as the command.

Reply:
Ahh. Did not know that.